src/options.ts

```diff
@@ -259,7 +260,13 @@ export async function loadOptions (configOverrides: NitroConfig = {}): Promise<N
   }
 
   // Resolve plugin paths
-  options.plugins = options.plugins.map(p => resolvePath(p, options))
+  options.plugins = options.plugins.map((p) => {
+    const path = resolvePath(p, options)
+    if (isWindows && options.dev && isAbsolute(path)) {
+      return pathToFileURL(path).href
```

Comment:
Would it broken on non windows using file urls?

Reply:
seems to work fine in the repro in one of the linked issues., but I seem to recall previously there were issues with file urls outside of windows environments (but that might have been in a nuxt/vite environment 🤔). if you think it better I am happy to remove the condition.

Reply:
I think it was vite since we already use file URLs for dev normally. Let's try with consistent behavior and limit to windows if was causing trouble